instrumentation/aws-sdk/aws-sdk-2.2/library/src/test/groovy/io/opentelemetry/instrumentation/awssdk/v2_2/Aws2ClientNotRecordHttpErrorTest.groovy

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.awssdk.v2_2
+
+import io.opentelemetry.instrumentation.test.LibraryTestTrait
+import software.amazon.awssdk.core.client.config.ClientOverrideConfiguration
+
+class Aws2ClientNotRecordHttpErrorTest extends AbstractAws2ClientRecordHttpErrorTest implements LibraryTestTrait {
```

Comment:
given this issue https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/7195, should we add these tests as Java code? Is there any good reason for not creating these tests as java code?

Reply:
I will convert the test to java